### PR TITLE
Model.integrate returns data and does not save by default, removed so…

### PR DESCRIPTION
**Pull request by Sosnowsky**
…me unused lines

### CODE REVIEW

File: blobmodel/model.py [hunk 1]
```diff
@@ -257,4 +253,13 @@ def integrate(self, file_name='2d_blobs.nc', speed_up=False, truncation_Lx=3):
             attrs=dict(description="2D propagating blobs."),
         )
 
-        ds.to_netcdf(file_name)
+        if file_name is not None:
+            ds.to_netcdf(file_name)
+
+        return ds
+
+    def get_blobs(self):
```
Comment (gregordecristoforo) [inline blobmodel/model.py:261]: This function is a good idea!

File: blobmodel/model.py [hunk 2]
```diff
@@ -215,7 +211,7 @@ def animate(i):
             ani.save(gif_name, writer='ffmpeg', fps=fps)
         plt.show()
 
-    def integrate(self, file_name='2d_blobs.nc', speed_up=False, truncation_Lx=3):
```
Comment (gregordecristoforo) [inline blobmodel/model.py:218]: I don't see the point in this either, why would you call integrate if you don't want to save the data?

**Review by gregordecristoforo — CHANGES_REQUESTED** [review]
@Sosnowsky

File: blobmodel/model.py [hunk 3]
```diff
@@ -183,9 +183,6 @@ def show_model(self, interval=100, save=False, gif_name='2d_blobs.gif', fps=10):
         fig = plt.figure()
         ax = fig.add_subplot(111)
```
[the remark this answers is not in the document]
Comment (Sosnowsky) [reply blobmodel/model.py:187]: fixed

**Review by gregordecristoforo — APPROVED** [review]

**Pull request merged**